documentation/asciidoc/accessories/camera/external_trigger.adoc

```diff
@@ -55,15 +44,35 @@ The low pulse width is equal to the shutter time, and the frequency of the PWM e
 
 NOTE: In this example, Pin 28 connects to the XTR touchpoint on the GS camera board.
 
-=== Operation
+=== Camera driver configuration
+
+This step is only necessary if you have more than one camera with XTR wired in parallel.
+
+As the superuser, edit `/boot/firmware/config.txt`. Change `camera_auto_detect=1` to `camera_auto_detect=0`.
+
+For either a source or a sink, append this line:
+[source]
+----
+dtoverlay=imx296,always-on
+----
+When using the CAM0 port on a Raspberry Pi 5, CM4 or CM5, append `,cam0` to that line without a space. If both cameras are on the same Raspberry Pi you will need two dtoverlay lines, only one of them with `cam0`.
+
+=== Starting the camera
+
+Enable external triggering through superuser mode:
```

Comment:
I think I'd shorten this to just "Enable external triggering:"

Reply:
Inherited from the original at L26, but can be shortened.

But now I'm wondering if I strictly need to edit  `external_trigger.adoc`?  The original was nice and succinct. The "always-on" parameter is perhaps not essential in this case because the driver is quite low impedance.